lib/_http_server.js

```diff
@@ -383,6 +384,7 @@ function connectionListenerInternal(server, socket) {
 
   // Override on to unconsume on `data`, `readable` listeners
   socket.on = socketOnWrap;
+  socket.setEncoding = socketSetEncoding;
```

Comment:
nit: either a comment or a new line here would be nice, since this line isn't related to

```js
  // Override on to unconsume on `data`, `readable` listeners
```